### PR TITLE
Sunrise FAQ updated changes.

### PR DESCRIPTION
**Pull request by idalithb**
Updated language

### CODE REVIEW

File: website/pages/en/sunrise.mdx [hunk 1]
```diff
@@ -74,15 +74,15 @@ This new version will then sync in Subgraph Studio, a testing and sandbox enviro
...
+When the owner has upgraded their subgraph, you will be able to easily go from the subgraph's hosted service page to the corresponding subgraph on The Graph Network, and update your application to use the new subgraph's query URL ([learn more](querying/querying-the-graph)).
 
-### I use a subgraph developed by someone else, how can I make sure that my service isn't interrupted?
+Around the start of June, Edge & Node will automatically upgrade actively queried subgraphs. This will give any third-party data consumers an opportunity to move subgraph endpoints to The Graph Network before June 12th. The subgraph owners will still be able to claim these subgraphs on the network using the hosted service upgrade flow.
 
-When the owner has upgraded their subgraph, you will be able to easily go from the subgraph's Hosted Service page to the corresponding subgraph on The Graph Network, and update your application to use the new subgraph's query URL ([learn more](querying/querying-the-graph)).
+### What happens if I don't upgrade my subgraph?
```
Comment (idalithb) [inline website/pages/en/sunrise.mdx:83]: This question will be updated to the following after June 12th.  (I will put in the PR request then) 

> I didn’t upgrade my subgraph by June 12th. What should I do if I still want to use it?
> 
> 
> The [hosted service](https://thegraph.com/hosted-service) homepage is still accessible and can be used to search for legacy hosted service subgraphs. If your hosted service subgraph has already been auto-upgraded, you may claim its network equivalent as the original owner. If your subgraph was not already auto-upgraded, you can still upgrade and publish it.

File: website/pages/en/sunrise.mdx [hunk 2]
```diff
@@ -12,15 +12,15 @@ This plan draws on many previous developments from The Graph ecosystem, includin
 
 ### What are the phases of the Sunrise?
 
-**Sunray**: Enable support for hosted service chains, introduce a seamless upgrade flow, offer a free plan on The Graph Network as well as simple payment options.  
+**Sunray**: Enable support for hosted service chains, introduce a seamless upgrade flow, offer a free plan on The Graph Network, and simple payment options.
```
Comment (benface) [inline website/pages/en/sunrise.mdx:15]: Is the last item in the enumeration missing a verb?

File: website/pages/en/sunrise.mdx [hunk 3]
```diff
@@ -38,23 +38,23 @@ To upgrade a hosted service subgraph, you can visit the subgraph dashboard on th
 2. Select the receiving wallet (the wallet that will become the owner of the subgraph).
 3. Click the "Upgrade" button.
 
-Once your subgraph is published, the [upgrade Indexer](#what-is-the-upgrade-indexer) will begin serving queries on it, so you can begin making queries immediately, once you have generated an API key ([learn more](/cookbook/upgrading-a-subgraph/#what-next)).
+Once your subgraph is published, the [upgrade Indexer](#what-is-the-upgrade-indexer) will begin serving queries on it. Once you have generated an API key, you can begin making queries immediately. ([learn more](/cookbook/upgrading-a-subgraph/#what-next)).
```
Comment (benface) [inline website/pages/en/sunrise.mdx:41]:
```suggestion
Once your subgraph is published, the [upgrade Indexer](#what-is-the-upgrade-indexer) will begin serving queries on it. Once you have generated an API key, you can begin making queries immediately. [Learn more](/cookbook/upgrading-a-subgraph/#what-next).
```

File: website/pages/en/sunrise.mdx (hunk 2 above)
Comment (idalithb) [reply website/pages/en/sunrise.mdx:15]: offers!

File: website/pages/en/sunrise.mdx [hunk 4]
```diff
@@ -38,23 +38,23 @@ To upgrade a hosted service subgraph, you can visit the subgraph dashboard on th
...
 
 ### How can I get support with the upgrade process?
 
 The Graph community is here to support developers as they move to The Graph Network. Join The Graph's [Discord server](https://discord.gg/vtvv7FP) and request help in the #upgrade-decentralized-network channel.
 
 ### How can I ensure high quality of service and redundancy for subgraphs on The Graph Network?
 
-All subgraphs will be supported by the upgrade indexer. Developers can add GRT signal to Subgraphs eligible for indexing rewards in order to attract additional Indexers, for redundancy and improved quality of service ([learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph))
+All subgraphs will be supported by the upgrade indexer. In order to attract additional Indexers for redundancy and improved quality of service, developers can add GRT signal to subgraphs eligible for indexing rewards ([learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph)).
```
Comment (benface) [inline website/pages/en/sunrise.mdx:49]:
```suggestion
All subgraphs will be supported by the upgrade indexer. In order to attract additional Indexers for redundancy and improved quality of service, developers can add GRT signal to subgraphs eligible for indexing rewards. [Learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph)).
```

File: website/pages/en/sunrise.mdx [hunk 5]
```diff
@@ -74,15 +74,15 @@ This new version will then sync in Subgraph Studio, a testing and sandbox enviro
 
 > Publishing requires Arbitrum ETH - upgrading your subgraph also airdrops a small amount to facilitate your first protocol interactions 🧑‍🚀
 
-### What happens if I don't upgrade my subgraph?
+### I use a subgraph developed by someone else, how can I make sure that my service isn't interrupted?
 
-Subgraphs will be queryable on the hosted service until the cutoff date, after which query endpoints will no longer be available, and owners will not be able to deploy new versions. Owners will still be able to simply upgrade their subgraphs to the network at any time, even after the cutoff date.
+When the owner has upgraded their subgraph, you will be able to easily go from the subgraph's hosted service page to the corresponding subgraph on The Graph Network, and update your application to use the new subgraph's query URL ([learn more](querying/querying-the-graph)).
```
Comment (benface) [inline website/pages/en/sunrise.mdx:79]:
```suggestion
When the owner has upgraded their subgraph, you will be able to easily go from the subgraph's hosted service page to the corresponding subgraph on The Graph Network, and update your application to use the new subgraph's query URL. [Learn more](querying/querying-the-graph).
```

File: website/pages/en/sunrise.mdx [hunk 6]
```diff
@@ -122,11 +122,11 @@ The upgrade Indexer offers a powerful opportunity for Delegators. As more subgra
 
 No, the upgrade Indexer will only allocate the minimum amount per subgraph and will not collect indexing rewards.
 
-It operates on an “as needed” basis, and serves as a fallback until sufficient service quality is achieved by at least 3 other Indexers in the network for respective chains and subgraphs.
+It operates on a “as needed” basis and serves as a fallback until sufficient service quality is achieved by at least 3 other Indexers in the network for respective chains and subgraphs.
 
 ### How will this affect subgraph developers?
 
-Subgraph developers will be able to query their subgraphs on the network almost immediately after upgrading from the hosted service or publishing from Subgraph Studio, as no lead time will be required for indexing.
+Subgraph developers will be able to query their subgraphs on the The Graph Network almost immediately after upgrading from the hosted service or publishing from Subgraph Studio, as no lead time will be required for indexing.
```
Comment (benface) [inline website/pages/en/sunrise.mdx:129]:
```suggestion
Subgraph developers will be able to query their subgraphs on The Graph Network almost immediately after upgrading from the hosted service or publishing from Subgraph Studio, as no lead time will be required for indexing.
```

File: website/pages/en/sunrise.mdx [hunk 7]
```diff
@@ -142,15 +142,15 @@ The upgrade Indexer will serve a subgraph until it is sufficiently and successfu
...
+Other Indexers are incentivized to support subgraphs with ongoing query volume. The query volume to the upgrade Indexer should trend towards zero, as it will have a small allocation size, and other Indexers will be chosen for queries ahead of it.
 
 ## About The Graph Network
 
 ### Do I need to run my own infrastructure?
 
 No, all infrastructure is operated by independent Indexers on The Graph Network, including the upgrade Indexer ([read more below](#what-is-the-upgrade-indexer)).
 
-You can use the [Subgraph Studio](https://thegraph.com/studio/) to create, test, and publish your subgraph. All hosted service users must upgrade their subgraph to The Graph Network before the end of the upgrade window ending on June 12th 2024.
+You can use the [Subgraph Studio](https://thegraph.com/studio/) to create, test, and publish your subgraph. All hosted service users must upgrade their subgraph to The Graph Network before June 12th, 2024.
```
Comment (benface) [inline website/pages/en/sunrise.mdx:153]:
```suggestion
You can use [Subgraph Studio](https://thegraph.com/studio/) to create, test, and publish your subgraph. All hosted service users must upgrade their subgraph to The Graph Network before June 12th, 2024.
```

File: website/pages/en/sunrise.mdx [hunk 8]
```diff
@@ -160,26 +160,26 @@ Once your subgraph has reached adequate curation signal and other Indexers begin
...
+If you are building in web3, the moment you use a centralized indexing provider, you are giving them control of your dapp and data. The Graph’s decentralized network offers [superior quality of service](https://thegraph.com/blog/qos-the-graph-network/), reliability with unbeatable uptime thanks to node redundancy, significantly [lower costs](/network/benefits/), and keeps you from being hostage at the data layer.
 
-With The Graph Network, your subgraph is public and anyone can query it openly, which increases the usage and network effects of your dapp. With a centralized indexing solution, the subgraph is private to the centralized provider.
+With The Graph Network, your subgraph is public and anyone can query it openly, which increases the usage and network effects of your dapp. However, with a centralized indexing solution, the subgraph is private to the centralized provider.
```
Comment (benface) [inline website/pages/en/sunrise.mdx:171]: I'm not sure the "however" works well here; to me it reads like we're going to list a bad effect of subgraphs being public.

File: website/pages/en/sunrise.mdx [hunk 9]
```diff
@@ -74,15 +74,15 @@ This new version will then sync in Subgraph Studio, a testing and sandbox enviro
 
 > Publishing requires Arbitrum ETH - upgrading your subgraph also airdrops a small amount to facilitate your first protocol interactions 🧑‍🚀
 
-### What happens if I don't upgrade my subgraph?
+### I use a subgraph developed by someone else, how can I make sure that my service isn't interrupted?
 
-Subgraphs will be queryable on the hosted service until the cutoff date, after which query endpoints will no longer be available, and owners will not be able to deploy new versions. Owners will still be able to simply upgrade their subgraphs to the network at any time, even after the cutoff date.
+When the owner has upgraded their subgraph, you will be able to easily go from the subgraph's hosted service page to the corresponding subgraph on The Graph Network, and update your application to use the new subgraph's query URL. [Learn more](querying/querying-the-graph).
```
Comment (benface) [inline website/pages/en/sunrise.mdx:79]:
```suggestion
When the owner has upgraded their subgraph, you will be able to easily go from the subgraph's hosted service page to the corresponding subgraph on The Graph Network, and update your application to use the new subgraph's query URL. [Learn more](/querying/querying-the-graph).
```

File: website/pages/en/sunrise.mdx [hunk 10]
```diff
@@ -74,15 +74,15 @@ This new version will then sync in Subgraph Studio, a testing and sandbox enviro
...
+Around the start of June, Edge & Node will automatically upgrade actively queried subgraphs. This will give any third-party data consumers an opportunity to move subgraph endpoints to The Graph Network before June 12th. The subgraph owners will still be able to claim these subgraphs on the network using the hosted service upgrade flow.
 
-When the owner has upgraded their subgraph, you will be able to easily go from the subgraph's Hosted Service page to the corresponding subgraph on The Graph Network, and update your application to use the new subgraph's query URL ([learn more](querying/querying-the-graph)).
+### What happens if I don't upgrade my subgraph?
 
-Around the start of June, Edge & Node will automatically upgrade actively queried subgraphs. This will give any third party data consumers an opportunity to move to the new Network subgraph endpoints before the cutoff date. The subgraph owners will still be able to “claim” these subgraphs on the network using the Hosted Service upgrade flow.
+Subgraphs will be quaryable on the hosted service until June 12th. Owners will still be able to upgrade their subgraphs to The Graph Network at any time, even after June 12th.
```
Comment (azf20) [inline website/pages/en/sunrise.mdx:85]: Why remove the information about deploying new versions?

File: website/pages/en/sunrise.mdx (hunk 8 above)
Comment (idalithb) [reply website/pages/en/sunrise.mdx:171]: I frankly don't think the whole sentence after "however" is needed at all. It only talks about the issue with centralized indexing, but I don't think it's needed here.

File: website/pages/en/sunrise.mdx (hunk 2 above)
Comment (idalithb) [inline website/pages/en/sunrise.mdx:15]:
```suggestion
**Sunray**: Enable support for hosted service chains, introduce a seamless upgrade flow, offer a free plan on The Graph Network, and provide simple payment options.  
```

File: website/pages/en/sunrise.mdx [hunk 11]
```diff
@@ -122,11 +122,11 @@ The upgrade Indexer offers a powerful opportunity for Delegators. As more subgra
 
 No, the upgrade Indexer will only allocate the minimum amount per subgraph and will not collect indexing rewards.
 
-It operates on an “as needed” basis, and serves as a fallback until sufficient service quality is achieved by at least 3 other Indexers in the network for respective chains and subgraphs.
+It operates on a “as needed” basis and serves as a fallback until sufficient service quality is achieved by at least 3 other Indexers in the network for respective chains and subgraphs.
```
Comment (azf20) [inline website/pages/en/sunrise.mdx:125]: isn't "an" correct here?

File: website/pages/en/sunrise.mdx [hunk 12]
```diff
@@ -160,26 +160,26 @@ Once your subgraph has reached adequate curation signal and other Indexers begin
...
+If you are building in web3, the moment you use a centralized indexing provider, you are giving them control of your dapp and data. The Graph’s decentralized network offers [superior quality of service](https://thegraph.com/blog/qos-the-graph-network/), reliability with unbeatable uptime thanks to node redundancy, significantly [lower costs](/network/benefits/), and keeps you from being hostage at the data layer.
 
-With The Graph Network, your subgraph is public and anyone can query it openly, which increases the usage and network effects of your dapp. With a centralized indexing solution, the subgraph is private to the centralized provider.
+With The Graph Network, your subgraph is public and anyone can query it openly, which increases the usage and network effects of your dapp. However, with a centralized indexing solution, the subgraph is private to the centralized provider.
 
-Additionally, the upgrade Indexer enables 100,000 free monthly queries for your subgraph before needing to pay for queries.
+Additionally, the upgrade Indexer enables 100,000 free monthly queries for your subgraph before payment is needed for additional queries.
```
Comment (azf20) [inline website/pages/en/sunrise.mdx:173]:
```suggestion
Additionally, Subgraph Studio provides 100,000 free monthly queries on the Free Plan, before payment is needed for additional usage.
```

**Review by benface — APPROVED** [review]
Great work!

**Review by azf20 — COMMENTED** [review]
Thanks - a couple of questions in-line

File: website/pages/en/sunrise.mdx (hunk 11 above)
Comment (idalithb) [reply website/pages/en/sunrise.mdx:125]:
```suggestion
It operates on an “as needed” basis and serves as a fallback until sufficient service quality is achieved by at least 3 other Indexers in the network for respective chains and subgraphs.
```

File: website/pages/en/sunrise.mdx (hunk 12 above)
Comment (idalithb) [reply website/pages/en/sunrise.mdx:173]: Should free plan be capitalized? 
Thank you--great catch!

File: website/pages/en/sunrise.mdx (hunk 8 above)
Comment (idalithb) [reply website/pages/en/sunrise.mdx:171]:
```suggestion
With The Graph Network, your subgraph is public and anyone can query it openly, which increases the usage and network effects of your dapp. 
```

File: website/pages/en/sunrise.mdx [hunk 13]
```diff
@@ -12,15 +12,15 @@ This plan draws on many previous developments from The Graph ecosystem, includin
...
+**Sunray**: Enable support for hosted service chains, introduce a seamless upgrade flow, offer a free plan on The Graph Network, and simple payment options.  
 **Sunbeam**: The upgrade window that subgraph developers will have to upgrade their hosted service subgraphs to The Graph Network. This window will end on June 12th 2024.  
 **Sunrise**: Hosted service endpoints will no longer be available after June 12th 2024.
 
 ## Upgrading subgraphs to The Graph Network
 
 ### When will hosted service subgraphs no longer be available?
 
-Hosted service subgraphs will no longer be available after June 12th, 2024 (the cutoff date). Developers will no longer be able to deploy new versions, and query endpoints will no longer be available to query, returning an error message explaining what to do next.
+Hosted service endpoints will no longer be available after June 12th, 2024.
```
[the remark this answers is not in the document]
Comment (idalithb) [reply website/pages/en/sunrise.mdx:23]:
```suggestion
Hosted service endpoints will no longer be available after June 12th, 2024. Developers will no longer be able to deploy new versions, and query endpoints will no longer be available to query.
```

File: website/pages/en/sunrise.mdx (hunk 10 above)
Comment (idalithb) [reply website/pages/en/sunrise.mdx:85]:
```suggestion
Subgraphs will be queryable on the hosted service until June 12th, after which query endpoints will no longer be available and owners will not be able to deploy new versions. Owners will still be able to upgrade their subgraphs to The Graph Network at any time, even after June 12th.
```

**Review by idalithb — COMMENTED** [review]
Made all suggested changes.

File: website/pages/en/sunrise.mdx [hunk 14]
```diff
@@ -38,23 +38,23 @@ To upgrade a hosted service subgraph, you can visit the subgraph dashboard on th
...
 
 ### How can I get support with the upgrade process?
 
 The Graph community is here to support developers as they move to The Graph Network. Join The Graph's [Discord server](https://discord.gg/vtvv7FP) and request help in the #upgrade-decentralized-network channel.
 
 ### How can I ensure high quality of service and redundancy for subgraphs on The Graph Network?
 
-All subgraphs will be supported by the upgrade indexer. Developers can add GRT signal to Subgraphs eligible for indexing rewards in order to attract additional Indexers, for redundancy and improved quality of service ([learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph))
+All subgraphs will be supported by the upgrade indexer. In order to attract additional Indexers for redundancy and improved quality of service, developers can add GRT signal to subgraphs eligible for indexing rewards. [Learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).
```
Comment (MichaelMacaulay) [inline website/pages/en/sunrise.mdx:49]: developers can add curation signal

**Review by MichaelMacaulay — APPROVED** [review]
Great work! Only one tiny improvement from me!

File: website/pages/en/sunrise.mdx (hunk 14 above)
Comment (idalithb) [reply website/pages/en/sunrise.mdx:49]:
```suggestion
All subgraphs will be supported by the upgrade indexer. In order to attract additional Indexers for redundancy and improved quality of service, developers can add curation signal to subgraphs eligible for indexing rewards. [Learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).
```

**Pull request merged**